smf/switch_zone_setup/manifest.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+
+<service_bundle type='manifest' name='switch_zone_setup'>
+
+<service name='system/illumos/switch_zone_setup' type='service' version='1'>
+  <create_default_instance enabled='false' />
+  <single_instance />
+
+  <dependency name='multi_user' grouping='require_all' restart_on='none'
+    type='service'>
+    <service_fmri value='svc:/system/filesystem/local:default' />
```

Comment:
I think you can remove this `filesystem/local` dependency here.  The `multi-user` milestone you're depending on is relatively coarse-grained, and itself always depends on having the local file system available before it is online.

Reply:
Done, thanks.